src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -141,6 +142,19 @@ Status MediapipeGraphExecutor::infer(const KFSRequest* request, KFSResponse* res
         SPDLOG_DEBUG("KServe request for mediapipe graph: {} initialization failed with message: {}", request->model_name(), absMessage);
         return Status(StatusCode::MEDIAPIPE_GRAPH_INITIALIZATION_ERROR, std::move(absMessage));
     }
+    // Passing whole KFS request and response
+    if (this->passKfsRequestFlag) {
+        if (this->config.output_stream().size() != 1) {
+            SPDLOG_ERROR("KServe request for mediapipe graph output size is incorrect for passing whole KFS request and response");
```

Comment:
```suggestion
            SPDLOG_ERROR("KServe passthrough through mediapipe graph requires having only one input (request) and one output(response)");
```
Anyway this is to late for this message. We should validate that in MediapipeGraphDefinition

Reply:
The same comment stands for input stream size

Reply:
Ok.